mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -452,29 +452,62 @@ LogicalResult PopulateParamsAccel::obtainTuningParameters(
 const InitParamsAccel
 PopulateParamsXDL::initParameters[PopulateParamsXDL::nInitParameters] = {
```

Comment:
You may need to update: `nInitParameters`, `nInitParametersFp16` and `nInitParametersForward8Bit` to match with the updated size.

Out of curiosity, the it might be able to compile getting rid of those hard-coded constants since this is all statically initialized and c++ compiler can figure out a size.